database/entity/0072-add_communityuuid_to_transactions_table/Transaction.ts

```diff
@@ -0,0 +1,163 @@
+/* eslint-disable no-use-before-define */
+import { Decimal } from 'decimal.js-light'
+import { BaseEntity, Entity, PrimaryGeneratedColumn, Column, OneToOne, JoinColumn } from 'typeorm'
+import { DecimalTransformer } from '../../src/typeorm/DecimalTransformer'
+import { Contribution } from '../Contribution'
+import { DltTransaction } from '../DltTransaction'
+
+@Entity('transactions')
+export class Transaction extends BaseEntity {
+  @PrimaryGeneratedColumn('increment', { unsigned: true })
+  id: number
+
+  @Column({ type: 'int', unsigned: true, unique: true, nullable: true, default: null })
+  previous: number | null
+
+  @Column({ name: 'type_id', unsigned: true, nullable: false })
```

Comment:
nullable: false is default can also be omitted.

Reply:
that's right...